include/sys/mempool_base.h

```diff
@@ -76,7 +72,7 @@ struct sys_mem_pool_base {
 
 /* How many bytes for the bitfields of a single level? */
 #define Z_MPOOL_LBIT_BYTES(maxsz, minsz, l, n_max)	\
-	(Z_MPOOL_LVLS((maxsz), (minsz)) > (l) ?		\
+	(Z_MPOOL_HAVE_LVL(maxsz, minsz, l) ?		\
```

Comment:
for consistency please keep macro arguments inside brackets e.g. `(maxsz)` and `(minsz)`

Reply:
also in line 42